pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -26,24 +27,29 @@ Side-effect/Warning
 
 `BALENA_SUPERVISOR_LOG_CONTROL`: Any logs written by the user container or the device Agent are not sent to the dashboard when this variable is set to False.
 
+`BALENA_SUPERVISOR_HARDWARE_METRICS`: [Device metrics][device-metrics] such as CPU temperature and memory usage won't be reported to the API, and won't be displayed in the device's dashboard summary page.
 
 Data usage impact
 -----------------
 
-| Service                                             | Usage (Including DNS overhead) | Control Variable               |
-|-----------------------------------------------------|--------------------------------|--------------------------------|
-| API poll (Once per 10m)                             | 6650 Bytes per request         | BALENA_SUPERVISOR_POLL_INTERVAL   |
-| {{ $names.company.lower }} supervisor update poll (Once every 24 hours)       | 6693 Bytes per request         | Not configurable via variables |
-| VPN enabled                                         | 43 Bytes / second              | BALENA_SUPERVISOR_VPN_CONTROL   |
-| TCP check cost (When VPN disabled)                  | 47.36 Bytes / second           | BALENA_SUPERVISOR_CONNECTIVITY_CHECK |
-
+| Service                                                                 | Usage (Including DNS overhead)                          | Related Configuration Variable       |
+|-------------------------------------------------------------------------|---------------------------------------------------------|--------------------------------------|
+| API poll (Once per 15min)                                               | 6650 Bytes per request                                  | BALENA_SUPERVISOR_POLL_INTERVAL      |
+| {{ $names.company.lower }} Supervisor update poll (Once every 24 hours) | 6693 Bytes per request                                  | Not configurable via variables       |
+| VPN enabled                                                             | 43 Bytes / second                                       | BALENA_SUPERVISOR_VPN_CONTROL        |
+| TCP check cost (When VPN disabled)                                      | 47.36 Bytes / second                                    | BALENA_SUPERVISOR_CONNECTIVITY_CHECK |
+| Hardware metrics reporting                                              | ~66 Bytes every 10 seconds when there have been changes | BALENA_SUPERVISOR_HARDWARE_METRICS   |
 
 Example minimum bandwidth settings
 ----------------------------------
 
-The following settings tune the data usage to approximately 1.3MB per month:
+The following settings result in data usage of approximately 1.3MB per month:
 
 * Disable VPN
 * Disable CONNECTIVITY CHECK
 * Change POLL interval to 24 hours
 * Disable LOGS
```

Comment:
Is there a reason why these 3 are capital, if they do symbolize a config var then please put them under backticks